ui/app/components/http-requests-container.js

```diff
@@ -0,0 +1,56 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import isWithinRange from 'date-fns/is_within_range';
+import addMonths from 'date-fns/add_months';
+
+/**
+ * @module HttpRequestsContainer
+ * The HttpRequestsContainer component is the parent component of the HttpRequestsDropdown, HttpRequestsBarChart, and HttpRequestsTable components. It is used to handle filtering the bar chart and table according to selected time window from the dropdown.
+ *
+ * @example
+ * ```js
+ * <HttpRequestsContainer @counters={counters}/>
+ * ```
+ *
+ * @param counters=null {Array} - A list of objects containing the total number of HTTP Requests for each month. `counters` should be the response from the `/internal/counters/requests` endpoint which looks like:
+ * COUNTERS = [
+ *    {
+ *       "start_time": "2019-05-01T00:00:00Z",
+ *       "total": 50
+ *     }
+ * ]
+ */
+
+export default Component.extend({
+  classNames: ['http-requests-container'],
+  counters: null,
+  timeWindow: 'All',
+  filteredCounters: computed('timeWindow', function() {
```

Comment:
Does `counters` also need to be in the computed 'dependency' here? I'm not massively sure myself?

Reply:
ah yes it does, thank you!